src/views/studio/CreateEditChannelView/CreateEditChannelView.style.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from '@emotion/styled'
+import { sizes, breakpoints, transitions } from '@/shared/theme'
+import { Textarea, ActionBarTransaction, Avatar, FormField } from '@/shared/components'
+import { TitleSection } from '@/views/consumer/ChannelView/ChannelView.style'
+
+type AtionBarProps = {
+  isActive?: boolean
+}
+
+export const StyledTitleSection = styled(TitleSection)`
+  display: inline-flex;
+  width: auto;
+  flex-direction: row;
+  padding-top: ${sizes(8)};
+  @media (min-width: ${breakpoints.small}) {
+    padding-top: 0;
+  }
+`
+
+export const TitleContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+`
+
+export const InnerFormContainer = styled.div`
+  width: 100%;
+  max-width: 760px;
```

Comment:
I'm a bit unsure about this and the `margin-left` below but I think those may be affected by the max view width we discussed in the chat so let's see how it goes